src/libsyntax/ext/deriving/generic/mod.rs

```diff
@@ -1048,15 +1048,23 @@ impl<'a> MethodDef<'a> {
     /// discriminant values.  See issue #15523.)
```

Comment:
I looks like this bug here was fixed, so it's a good time to update this comment too I think. I assume using discr value is always correct now?